### DISCUSSION TITLE
serving-static-content

### DISCUSSION THREAD

**Discussion by jadsongmatos** [NginxProxyManager/nginx-proxy-manager#1397] [Ideas]
[https://docs.nginx.com/nginx/admin-guide/web-server/serving-static-content/](https://docs.nginx.com/nginx/admin-guide/web-server/serving-static-content/)

does this feature already exist ?

**Comment by chaptergy** [comment]
It does not, but there is an open feature request for this: https://github.com/jc21/nginx-proxy-manager/issues/58
